src/components/include/application_manager/application_manager.h

```diff
@@ -153,9 +157,40 @@ class ApplicationManager {
   virtual ApplicationSharedPtr application_by_policy_id(
       const std::string& policy_app_id) const = 0;
 
-  virtual std::vector<ApplicationSharedPtr> applications_by_button(
-      uint32_t button) = 0;
-  virtual std::vector<ApplicationSharedPtr> applications_with_navi() = 0;
+  virtual AppSharedPtrs applications_by_button(uint32_t button) = 0;
+  virtual AppSharedPtrs applications_with_navi() = 0;
+
+#ifdef SDL_REMOTE_CONTROL
+  virtual ApplicationSharedPtr application(
```

Comment:
please add description for the new functions

Reply:
@LuxoftAKutsan application() still needs function desription